### DISCUSSION TITLE
Article about "Holistic Observability with OpenTelemetry and OpenObserve"

### DISCUSSION THREAD

**Discussion by r0b2g1t** [openobserve/openobserve#1359] [Show and tell]
Hello,

I have written an article about how to use OpenObserve in combination with an OpenTelemetry Collector. That includes the setup, configuration and a basic insight to OpenObserve. Maybe this is interesting for someone. 😊

[https://medium.com/itnext/holistic-observability-with-opentelemetry-and-openobserve-f599af459fbe](https://medium.com/itnext/holistic-observability-with-opentelemetry-and-openobserve-f599af459fbe)

**Comment by prabhatsharma** [comment]
good article
